election-server/src/bin/http.rs

```diff
@@ -256,14 +320,24 @@ async fn main() -> anyhow::Result<()> {
         )
         .with_state(state)
         .route_service(
-            "/static/eligible-voters.json",
-            ServeFile::new(config.eligible_voters_file),
+            "/static/concordium/eligible-voters.json",
+            ServeFile::new(config.eligible_voters_file.clone()),
```

Comment:
```suggestion
            ServeFile::new(&config.eligible_voters_file),
```

Reply:
Nice, applied for all similar occurrences!